kratos/processes/assign_scalar_variable_to_conditions_process.h

```diff
@@ -10,8 +10,8 @@
 //
 //  Main authors:    Josep Maria Carbonell
 //
-#if !defined(KRATOS_ASSIGN_SCALAR_VARIABLE_TO_CONDITIONS_PROCESS_H_INCLUDED )
```

Comment:
why are u changing this?

Reply:
This has conflicts wit the solid app,

Reply:
Not in this case, when you copy and modify a class from another app. If you do not change the name you create conflicts with the application which will not be compatible anymore.